packages/blocks/drawing-canvas/README.md

```diff
@@ -0,0 +1,94 @@
+# Block template
```

Comment:
Let's remove all the template contents. We can just add something like

```
# Drawing canvas
```

A couple of sentences with upstream links would be also useful.